src/desktop/apps/article/components/InfiniteScrollArticle.js

```diff
@@ -155,6 +160,7 @@ export default class InfiniteScrollArticle extends React.Component {
               showTooltips={showTooltips}
               showToolTipMarketData={showToolTipMarketData}
               onOpenAuthModal={onOpenAuthModal}
+              renderTime={renderTimes[i % 3]}
```

Comment:
@kanaabe Are you using `% 3` cause we recycle ads every 1/3 time?

Reply:
This is to get the right index of the renderTime but now that I think of it, it's not the right math. 1 sec..

Reply:
Ah this is what we want: `Math.floor(i / 3)`. Updated in the PR!